deepliif/postprocessing.py

```diff
@@ -250,11 +250,11 @@ def create_final_segmentation_mask_with_boundaries(mask_image):
     refined_mask = mask_image.copy()
 
     edges = feature.canny(refined_mask[:, :, 0], sigma=3).astype(np.uint8)
-    contours, _ = cv2.findContours(edges, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)
+    contours, _ = cv2.findContours(edges, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)[-2:] # a more recent cv2 version has 3 returned values
```

Comment:
Is there a specific reason for updating cv2 to a more recent version?

Reply:
@ricdodds no (it's simply that our environment uses a newer version, and downgrading it is more complicated than downgrading a regular package like pandas, maybe because of it os dependencies?); this change is compatible for both new and old cv2